### ISSUE TITLE
Windows Cursor Color

### ISSUE THREAD

**Issue by enisn** [enisn/UraniumUI#791]
### Discussed in https://github.com/enisn/UraniumUI/discussions/789

<div type='discussions-op-text'>

<sup>Originally posted by **pouchbunny** October  2, 2024</sup>
Can the cursor color be a bindable prop so we can set it to match the theme, or have it based on the Accent Color since it's the color being used when focusing on a textfield? When you put the text field on a frame the color of the cursor changes on its own and with my chosen color it's hard to see.

![image](https://github.com/user-attachments/assets/3dbe1ec2-86b2-4e26-a05d-bbf9e8373b01)

![image](https://github.com/user-attachments/assets/a4abae40-e987-4b94-a62f-603c671c6976)
</div>

**Comment by carmen-cruzarana** [comment]
May I help with this issue?

**Comment by enisn** [comment]
> May I help with this issue?

Of course, this issue is marked as [good first issue](https://github.com/enisn/UraniumUI/labels/good%20first%20issue) and PR is always welcome
